Program_Licenses.md

```diff
@@ -29,6 +29,7 @@ The licenses of the open-source software that is contained in these Docker image
 | cfsan-snp-pipeline | non-standard license see --> | https://github.com/CFSAN-Biostatistics/snp-pipeline/blob/master/LICENSE.txt |
 | CheckM | GNU GPLv3 | https://github.com/Ecogenomics/CheckM/blob/master/LICENSE |
 | Circlator | GNU GPLv3 | https://github.com/sanger-pathogens/circlator/blob/master/LICENSE |
+| Circos | GNU GPLv3 | https://circos.ca/ |

```

Comment:
For future reference, the license is at the bottom of this page. 

> LICENSE AND USE
Circos is free software, licensed under [GPL](https://www.gnu.org/copyleft/gpl.html).
> Circos is written in Perl, can be deployed on any operating system for which Perl is available (e.g. Windows, Mac OS X, Linux and other UNIX flavours) and produces bitmap (PNG) and vector (SVG) images using plain text configuration and input files.